CMakeLists.txt

```diff
@@ -2139,7 +2159,42 @@ if(QT5_TYPE STREQUAL "STATIC_LIBRARY")
 
 endif()
 
-if(UNIX AND NOT APPLE)
+
+if(APPLE)
+  if(QT5_TYPE STREQUAL "STATIC_LIBRARY")
+    find_package(FFTW REQUIRED)
+    find_package(SndFile REQUIRED)
+    find_package(sord CONFIG REQUIRED)
+    find_library(SAMPLERATE_LIBRARY samplerate REQUIRED)
```

Comment:
I'm confused... Mixxx doesn't use libsamplerate?

Reply:
rubberband uses libsamplerate

Reply:
...and Rubberband also needs FFTW, depending on the build config. Rubberband 1.9.2 comes with its own, optimized FFT code that doesn't require external dependencies.